com.mongodb.Compass.json

```diff
@@ -0,0 +1,88 @@
+{
+    "app-id": "com.mongodb.Compass",
+    "base": "org.electronjs.Electron2.BaseApp",
+    "base-version": "21.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.node14"
+    ],
+    "command": "run.sh",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--talk-name=org.freedesktop.secrets",
+        "--share=network"
+    ],
+    "modules": [
+        "shared-modules/libsecret/libsecret.json",
+        {
+            "name": "run",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm755 -t /app/bin/ run.sh"
+            ],
+            "build-options": {
+                "env": {
+                    "ZYPAK_SANDBOX_FILENAME": "chrome-sandbox",
+                    "ZYPAK_DEBUG": "1"
+                }
+            },
+            "sources": [
+                {
+                    "type": "script",
+                    "dest-filename": "run.sh",
+                    "commands": [
+                        "zypak-wrapper '/app/main/MongoDB Compass'"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "compass",
+            "buildsystem": "simple",
+            "build-commands": [
+                "cp -r main /app/main",
+                "install -Dm 644 icon-64x64.png /app/share/icons/hicolor/64x64/apps/com.mongodb.Compass.png",
```

Comment:
note that we only require 128x128 (or SVG). And given they are not provided upstream, I'd be enclined to says 128x128 is the only one we need to have.

Reply:
not what this says:
https://github.com/flathub/flathub/wiki/App-Requirements#application-icons

btw I have included the two required sized plus the original icon untouched

Reply:
yeah **64x64 only** is no longer accepted. But I can't edit the wiki, so it's just a lie now.